src/Tests/SenseNet.Tests/TestBase.cs

```diff
@@ -65,6 +65,10 @@ private void EnsurePrototypes()
         [TestInitialize]
         public void InitializeTest()
         {
+            // workaround for having a half-started repository
+            if (RepositoryInstance.Started())
+                RepositoryInstance.Shutdown();
```

Comment:
Should we put this inside a try-catch block?

Reply:
I think it's better if the exception fires because we get information about any problem.